CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes for each version of this project will be documented in this file.
 ## 5.3.0
+- Added `rowSelectable` property to `igxGrid`
+    - Setting `rowSelectable` to `true` enables multiple row selection for the `igx-grid` component. Adds a checkbox column that allows (de)selection of one, multiple or all (via header checkbox) rows.
+    - Selection is persistent through data virtualization, paging and all data operations (filtering, sorting).
+    - Added methods for (de)selection of rows in a `igx-grid`:
```

Comment:
Just provide a link to the readme of the feature. No need to describe the API here.

Reply:
Thanks for clarifying! I've added a link to the readme file and the (not yet available) documentation page in the latest commit.